sympy/concrete/expr_with_limits.py

```diff
@@ -1,7 +1,9 @@
 from __future__ import print_function, division
 
-from sympy.core.basic import C
 from sympy.core.expr import Expr
+from sympy.core.add import Add
+from sympy.core.mul import Mul
+from sympy.core.relational import Equality
```

Comment:
Can you keep the import list sorted?
That makes it easier for future code reading.